src/mame/nintendo/n64_lodgenet.cpp

```diff
@@ -0,0 +1,194 @@
+// license:BSD-3-Clause
+// copyright-holders:FoxhackDN, Angelo Salese
+/*
+    Nintendo Gateway 64
+
+    <fill me>
+
+    TODO:
+    - Set-top box part;
+    - Extra joypad buttons;
+    - Doesn't really have cartslot but rather uploads games thru a DOS/V Windows 95/98 program
+      that works with encrypted files.
+      \- Notice that the available dump is "patched to work on emulators",
+         it may normally expect a specific ISA/PCI board setup plus a setup/install CD-ROM;
```

Comment:
Given this and previous @cuavas concerns I'm very inclined to just c&p the cartslot stuff and derive from n64 base state instead (given that it eventually need to be removed if/when the real PC interface is working). 
It also needs those i64 files to actually be defined, something I'm currently not sure about how to do in XML form.